js/commerce_pos.js

```diff
@@ -140,4 +140,89 @@
       .trigger('blur');
   }
 
+  Drupal.behaviors.commercePosSalePay = {
```

Comment:
Oh I see ... we're renaming a js file to be more generic. What do you think about this change, @smmccabe ?